src/systemctl/systemctl.c

```diff
@@ -983,6 +983,14 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_WHEN:
+                        if (streq(optarg, "show")) {
+                                r = logind_show_shutdown();
+                                if (r < 0 && r != -ENODATA)
+                                        return r;
+
+                                return 0;
+                        }
```

Comment:
you could also accept "cancel" here as special string to cancel the thing again.

would be good to add a test case somewhere that guarantees that "show" and "cancel" do not parse as valid timestamps. just in case the timestamp syntax is extended one day, so that the CI fails immediately, should "cancel" or "show" suddenly be accepted timestamps.